sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobAPITest.groovy

```diff
@@ -716,7 +715,7 @@ class BlobAPITest extends APISpec {
         setup:
         def copyDestBlob = cc.getBlobClient(generateBlobName()).asBlockBlobClient()
         def headers =
-            copyDestBlob.startCopyFromURLWithResponse(bc.getBlobUrl(), null, null, null, null, null, null, null).getHeaders()
+            copyDestBlob.startCopyFromURLWithResponse(new URL(bc.getBlobUrl()), null, null, null, null, null, null, null).getHeaders()
```

Comment:
If this method requires a URL, then there should be a follow-up change (either here or in a separate PR) to changes these method calls to also take a String url.

Reply:
This is an interesting case as the Swagger interface is expecting a URL parameter, I'm wondering if it could make more sense as a String otherwise this method will need to handle the creation of the URL.

Reply:
Have created this issue to track this - #5656

Reply:
I am fine with tackling this in a follow-up issue